tests/bluetooth/tester/src/gatt.c

```diff
@@ -214,11 +214,20 @@ static void supported_commands(u8_t *data, u16_t len)
 
 static int register_service(void)
 {
+	int err;
+
 	server_svcs[svc_count].attrs = server_db +
 				       (attr_count - svc_attr_count);
 	server_svcs[svc_count].attr_count = svc_attr_count;
 
-	return bt_gatt_service_register(&server_svcs[svc_count]);
+	err = bt_gatt_service_register(&server_svcs[svc_count]);
+
```

Comment:
minor: remove this empty line (no empty line between assignment to a variable and testing its value)

Reply:
fixed, thanks